conceptpower/classes.py

```diff
@@ -157,4 +159,65 @@ def get_type(self, uri):
                 data['supertype_id'] = snode.get('supertype_id')
                 data['supertype_uri'] = snode.get('supertype_uri')
         return data
-        
\ No newline at end of file
+
+    def create(self, user, password, word, pos, conceptlist, description, type, synonym_ids, equal_uris, similar_uris):
+        """
+        Adds a concept to the conceptlist by using rest api
+
+        Parameters
+        -----------
+        user : str
+             userId of the person who wants to add concept
+        password : str
+                 password of the person who wants to add concept
+        word : str
+             name of the word
+        pos : str
+            part of speech of the word
+        conceptlist : str
+                    name of the concept list it belongs to
+        description : str
+                    description of the word
+        type : str
+             type of the word
+
+        synonymids : list
+                   ids of synonyms for the new concept
```

Comment:
On line 185, it looks like the description is indented more than 4 spaces further than the previous line